_events/2022/2022-03-dei-diverse-leung.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Normalizing inclusion by embracing difference
+subtitle: DiveRSE Speaker Series
+expires: 2022-03-23
+event_date: "March 22, 2022"
+layout: event
+duration: 60
+repeated: false
+category: dei 
+time:
+    - - start: 2022-03-11T16:00:00Z
+        end: 2022-03-11T17:30:00Z
+---
+
+## DiveRSE Speaker Series Info
+
+DiveRSE (Diverse RSE) is a series of talks and related activities including discussion and panel sessions designed to support and raise awareness of EDI challenges and successes within the research software community.
+
+## Event - Normalizing inclusion by embracing difference
+
+DiveRSE is delighted to welcome Mary Ann Leung, Ph.D., Founder and President of the Sustainable Horizons Institute, as our inaugural speaker in the DiveRSE talk series looking at diversity, equity and inclusion in the Research Software Engineering (RSE) community.
```

Comment:
I think there is a link for DiveRSE now?

Reply:
Yup! https://diverse-rse.github.io/